_layouts/eip.html

```diff
@@ -136,3 +136,6 @@ <h2>Citation</h2>
     "copyrightYear": "{{ page.created | date: "%Y" }}"
   }
 </script>
+<script type="application/javascript">
+  document.getElementsByClassName('footnotes')[0].insertAdjacentHTML('beforebegin', '<h2 id="footnotes">Footnotes</h2>');
```

Comment:
Won't this error if there are no footnotes?

Reply:
You're right. Let me fix that.